integration/README.md

```diff
@@ -0,0 +1,47 @@
+# Quick Base Craft Demo | Senior Integration Specialist
+
+## Craft Demo Presentation Outline
+
+We have created a PowerPoint template which covers an outline of what we would like to talk about during the first section of your interview before you share your craft demo. Feel free to use the template provided or make your own. We are more interested in your code than your PowerPoint skills, so plan your prep time accordingly.
+
+[PowerPoint Template and Craft Demo Outline](https://github.com/QuickBase/interview-demos/blob/master/QuickBase_CraftDemo_PresentationTemplate.pptx)
+
+## General Description
+
+The purpose of the craft demo is to give us a jumping-off-point for discussion of your approach to solving a problem and of your code. We are interested in learning how you approach this problem and go about solving it.
+
+We expect this project to take experienced developers about two-four hours to complete. We know not everyone has unlimited time to work on interview projects and we do not expect that everyone completes all of the requirements. However, you may be asked how you might have solved a requirement you missed had you been able to work on this project without time constraints. Additionally, if you want to extend the project beyond the listed requirements (e.g., tracking additional data points in the 'Tweet Tracker' app), please feel free to do so as you see fit. Please note that we prefer higher-code quality over additional features if you start to run out of time.
+
+Use of your favorite libraries, SDKs and frameworks is welcomed but optional. The project outlined below can be completed with or without these tools.
+
+Please be prepared to demo your solution in a working development environment on the day of your interview. When you present your craft demo, we will ask you to describe how you arrived at your solution. After that, we will ask for a tour of your code with a focus on how you arrived at the core functionality and any design decisions and/or tradeoffs you made during the process.
+
+## Demo Project Outline
+
+The Quick Base marketing team has a launched a brand-awareness campaign that is running for the next year and you have been tasked with tracking tweets about the company for the length of this engagement. You have been asked specifically to search Twitter for tweets that contain hashtags from a list compiled by our marketing team and store them in a Quick Base app for future investigation by our business analysts. 
+
+## Demo Project Requirements
+
+1. Your code should collect 100 or more tweets from 2018 that include the following hashtags:
+    * #QuickBase
+    * #NoCode
+    * #LowCode
+    * #QBEMPOWER18
```

Comment:
Remove the dated one so we don't have to remember to update this every year.

Reply:
All set